web/app/models/DrivingSession.js

```diff
@@ -0,0 +1,14 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our driving session model
+// module.exports allows us to pass this to other files when it is called
+var DrivingSessionSchema = new mongoose.Schema({
+    startTime: { type: Number, required: true },
+    endTime: { type: Number, required: true },
+    distance: { type: Number, required: true },
+    duration: { type: Number, required: true },
```

Comment:
Ah that's right points. A new issue should be open then for this. The schema is the final representation of the data upon storage. New functionality will need to be added that will process the received points on mobile for calculating distance.